commands/message.go

```diff
@@ -221,6 +223,82 @@ var msgWaitCmd = &cmds.Command{
 	},
 }
 
+// MessageStatusResult is the status of a message on chain or in the message queue/pool
+type MessageStatusResult struct {
+	InPool    bool
+	PoolMsg   *types.SignedMessage
+	InOutbox  bool
```

Comment:
this is a confusing name, but I can't think of anything better. A comment would help since I had to read all the way down to know what it means.